phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/CreateShortcutPanel.java

```diff
@@ -106,7 +106,7 @@ public void populate() {
 
         TextField miniature = new TextField();
 
-        Button openMiniatureBrowser = new Button(tr("Choose"));
+        Button openMiniatureBrowser = new Button(tr("Browse…"));
```

Comment:
See above.

Reply:
https://transvision.mozfr.org/?recherche=Browse%E2%80%A6&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings

Reply:
This only means that this is the way Mozilla does it. It does not mean that it is a general rule. We can discuss if we want to use this style or not but I would like to separate this from this pull request. Please open a new issue for it. The reason for this is: Like I said above, we should follow a consistent style in the software and the scripts. So if we change the software now, we must also change all the scripts and that's a lot of work I'm not willing to do right now.

Reply:
done

Reply:
...